starting.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+require_relative 'app'
+
+class Start
+  app = App.new
+
+  response = nil
+
+  puts "Welcome to School Library App!\n\n"
+  while response != '7'
+    puts 'Please choose an option by enter in a number from 1 to 7:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given id'
+    puts '7 - Exit'
+    response = gets.chomp
+
+    case response
+    when '1'
+      app.list_all_books
+    when '2'
+      app.list_all_people
+    when '3'
+      app.create_a_person
+    when '4'
```

Comment:
- [ ] In the application, when you are creating a Teacher, the input shouldn't have permissions based on the video requirements. Kindly change this

https://www.youtube.com/watch?v=vkkgrhD6aXQ

#### I would advise you to follow the video to know what should be printed out for the user. 👍